### PR TITLE
Tf2.0 tegra downstream rebase 1.7.19

### PR DESCRIPTION
**Pull request by vwadekar**
Tegra downstream rebase 1.17.19

### CODE REVIEW

File: plat/nvidia/tegra/common/tegra_bl31_setup.c [hunk 1]
```diff
@@ -315,35 +313,35 @@ void bl31_plat_runtime_setup(void)
...
+	uint32_t coh_start, coh_size;
 #endif
-	plat_params_from_bl2_t *params_from_bl2 = bl31_get_plat_params();
+	const plat_params_from_bl2_t *params_from_bl2 = bl31_get_plat_params();
 
 	/* add memory regions */
 	mmap_add_region(rw_start, rw_start,
 			rw_size,
-			MT_MEMORY | MT_RW | MT_SECURE);
+			(uint8_t)MT_MEMORY | (uint8_t)MT_RW | (uint8_t)MT_SECURE);
```
Comment (antonio-nino-diaz-arm) [inline plat/nvidia/tegra/common/tegra_bl31_setup.c]: I think that the casts here are incorrect, the attributes are ``unsigned int``. Same all over the file.

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

**Comment by vwadekar** [comment]
@antonio-nino-diaz-arm 

Although the checkpatch script doesn't run as expected at my end, was able to fix the issues that were flagged as part of this PR

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Pull request merged**